packages/cms-lib/fileMapper.js

```diff
@@ -265,29 +265,12 @@ async function fetchAndWriteFileStream(input, srcPath, filepath) {
     throw new Error(message);
   }
   const { portalId } = input;
-  const logFsError = err => {
-    logFileSystemErrorInstance(
-      err,
-      new FileSystemErrorContext({
-        filepath,
-        portalId,
-        write: true,
-      })
-    );
-  };
-  let writeStream;
-  try {
-    await fs.ensureFile(filepath);
```

Comment:
I don't believe we need this, because `createWriteStream` will create the file by default